src/node_jsvmapi.cc

```diff
@@ -864,6 +888,10 @@ napi_value napi_get_true(napi_env e) {
              v8::True(v8impl::V8IsolateFromJsEnv(e)));
 }
 
+napi_value napi_new_value(napi_env env) {
```

Comment:
So.. does this make an _empty_ value? If so, should it be named `napi_new_empty_value`?